Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -202,7 +202,7 @@ return
 							Type = "string",
 						},
 					},
-					Notes = "Returns the name of the sound that is played when placing the block of this type.",
+					Notes = "(<b>DEPRECATED</b>)",
```

Comment:
If there's no way for the plugins to change what sounds are played, then say so here.